tests/Sniffs/NamingConventions/Fixtures/CamelCaseVariableSniff/with_errors/NotCamelCase.11,15-17.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace BestIt\Sniffs\NamingConventions;
+
+class SuppressedClass {
+    /**
+     * A tet var.
+     *
+     * @var string
+     */
+    public $test_pro = 'foo';
+
+    private function testMethod()
+    {
+        $test_var = $this->test_pro;
```

Comment:
I would expect that this line does not throws an error - only at decleration at line 11. What happens if you surpress line 11? Does Codesniffer ignore this line or will throw an error? It would be uncool, if we must surpress every line instead just the line at decleration.

Reply:
@b3nl

Reply:
But $test_var is not valid! ;) the part after the equal is - as you „required“ ;) - not the reason for this error.

Reply:
Ah ok